src/app/state/hooks.js

```diff
@@ -61,6 +61,25 @@ export const useViewConfigStore = create(set => ({
   }),
 }));
 
+/**
+ * The useViewConfigStore hook is initialized via the zustand
+ * create() function, which sets up both the state variables
+ * and the reducer-type functions.
+ * Reference: https://github.com/react-spring/zustand
+ * @returns {function} The useStore hook.
+ */
+export const useAuxiliaryStore = create(set => ({
+  auxiliaryStore: null,
```

Comment:
Can you add comments here explaining what the `auxiliaryStore` is intended to be used for?